### PR TITLE
feature: Added Rejected Status to Reference Table

### PR DESCRIPTION
**Pull request by Snogglespike**
- Added rejected status to the reference table
- Added basic rejected label to reference screen

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -138,6 +138,7 @@ model Reference {
   hasKnowApplicantForAYear Boolean?
   isRelated                Boolean?
   isMentorRecommended      Boolean?
+  isRejected               Boolean?
```
Comment (duvet86) [inline prisma/schema.prisma:141]: Hi Tristan, the table has a `isMentorRecommended` column. What about leveraging that column?

**Review by duvet86 — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (Snogglespike) [reply prisma/schema.prisma:141]: Sounds good, I have a few ideas. Will play around with some things.

**Review by duvet86 — APPROVED** [review]
Perfect thanks.

**Review by duvet86 — APPROVED** [review]

**Pull request merged**